x-pack/plugins/alerting/server/task_runner/rule_action_helper.ts

```diff
@@ -69,7 +69,7 @@ export const isSummaryActionThrottled = ({
 };
 
 export const generateActionHash = (action: RuleAction) => {
-  return `${action.actionTypeId}:${action.group || 'summary'}:${
+  return `${action.actionTypeId}:${action.frequency?.summary ? 'summary' : action.group}:${
```

Comment:
nit: I would have expected a unit test update for this change but I see that the unit test defined `group: null` to achieve the desired hash. Can we update the unit test to reflect that the action group won't be null in reality?